kafka-sample/java/README.md

```diff
@@ -23,26 +23,28 @@ you can use the gradle `shadowJar` task to build an executable [jarfile](build/l
 gradle shadowJar
 ```
 
+
+Our Kafka server uses Let’s Encrypt CA for SSL/TLS certificates. These certificates are usually included in the default 
```

Comment:
How is our? Please use something like "In this example we assume to use Lets Encrypt..."

Reply:
implemented as suggested.